cspell.yml

```diff
@@ -0,0 +1,41 @@
+# Configuration for the cspell command-line tool and the Code Spell Checker
+# VSCode extension
+# (https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker).
+
+# Add a repo-specific dictionary. Any new real words can be added to
+# cspell-dict.txt, one per line.
+dictionaryDefinitions:
+  - name: workspace
+    path: './cspell-dict.txt'
+    description: Custom Workspace Dictionary
+    addWords: true
+    scope: workspace
+dictionaries:
+  - workspace
+
+allowCompoundWords: true
+
+# Ignore files that aren't checked in to git as well as files that aren't written
+# by hand. Note that we do want to check, say, JSON files (as package.json
+# contains English text like package descriptions).
+useGitignore: true
+ignorePaths:
+  - cspell.yaml
```

Comment:
Did you want the ignore path to match this file? Because if so => cspell.yml.

Reply:
Good catch, maybe it never mattered